### PR TITLE
Add rspec rubocop and fix code to pass checks

### PR DESCRIPTION
**Pull request by mcass19**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,17 +39,23 @@ gem 'jbuilder', '~> 2.5'
...
+  # a debugger console
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'rspec-rails', '~> 3.8.0'
 end
 
 group :development do
-  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
-  gem 'web-console', '>= 3.3.0'
+  # Access an interactive console on exception pages or by calling
+  # 'console' anywhere in the code.
```
Comment (rodrigopdl) [inline Gemfile]: I think the `gem 'web-console'` line was meant to be under this comment.

File: Gemfile [hunk 2]
```diff
@@ -37,17 +39,23 @@ gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  # Call 'byebug' anywhere in the code to stop execution and get
+  # a debugger console
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'rspec-rails', '~> 3.8.0'
```
Comment (rodrigopdl) [inline Gemfile]: `rspec-rails` documentation [suggests](https://github.com/rspec/rspec-rails#installation) going with `~> 3.8`. What's the reasoning behind going with `~> 3.8.0` instead?

File: Gemfile [hunk 3]
```diff
@@ -37,17 +39,23 @@ gem 'jbuilder', '~> 2.5'
...
 end
 
 group :development do
-  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
-  gem 'web-console', '>= 3.3.0'
+  # Access an interactive console on exception pages or by calling
+  # 'console' anywhere in the code.
   gem 'listen', '>= 3.0.5', '< 3.2'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'rails_best_practices', '~> 1.19.0'
```
Comment (rodrigopdl) [inline Gemfile]: Is this gem required to run `rubocop`? What problem does it solve?

**Comment by mcass19** [comment]
@rodrigopdl updated !

**Comment by mcass19** [comment]
@rodrigopdl Thanks for the comments ! I change the base branch of the PR to `master` and fix the bin files in 1ba6f2a.

**Comment by mcass19** [comment]
@rodrigopdl Thanks for the comments ! I change the base branch of the PR to master and fix the bin files in 179b154.

**Pull request merged**